src/main/java/com/epam/izh/rd/online/entity/Book.java

```diff
@@ -16,5 +19,54 @@
  * 6) Переопределить метод toString с выводом всех полей (не забывайте alt+inset)
  */
 public abstract class Book {
+    private int numberOfPages;
```

Comment:
Also you need id field

Reply:
ok, I see. But for now I just followed instructions. I guess when it will be the topic jpa or Hibernate - id will definitely be needed. At this time it is just another column which add "complexity".